recipes/qt/notes.md

```diff
@@ -0,0 +1,27 @@
+Packages needed in Ubuntu:
+
+- g++-4.4
+- libx11-dev
+- libxt-dev
+- libxext-dev
+- libfreetype6-dev
+- libfontconfig1-dev
+- chrpath
+- libxrender-dev
+- freeglut3-dev
+- libgtk2.0-dev (to enable gtkstyle)
+- libdbus-1-dev
+
+Packages needed in CentOS:
+
+- gcc-c++
+- libX11-devel
+- libXt-devel
+- libXext-devel
+- freetype-devel
+- fontconfig-devel
+- chrpath
+- libXrender-devel
+- freeglut-devel
+- gtk2-devel
+- dbus-devel
```

Comment:
Please add the stuff we don't have to a `yum_requirements.txt` file and at it with your recipe. Basically X11, GTK, and similar.